### PR TITLE
New URLDownloader using Kangooroo

### PR DESCRIPTION
**Pull request by gdesmar**

### CODE REVIEW

File: tests/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+assemblyline-service-utilities
```
Comment (cccs-kevin) [inline tests/requirements.txt:1]: don't need this

File: urldownloader.py [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+                                {
+                                    "status": entry["response"]["status"],
+                                    "redirecting_url": entry["request"]["url"],
+                                    "redirecting_ip": entry["serverIPAddress"],
+                                    "redirecting_to": refresh[1][4:],
+                                }
+                            )
+
+                    except Exception:
+                        # Maybe log that we weren't able to parse the refresh
```
Comment (cccs-kevin) [inline urldownloader.py:171]: maybe?

**Review by cccs-kevin — APPROVED** [review]
In general, some more comments in Dockerfile and urldownloader.py would be appreciated, but are not necessary

File: urldownloader.py (hunk 2 above)
Comment (gdesmar) [reply urldownloader.py:171]: maybe not. :D

**Review by cccs-kevin — APPROVED** [review]

**Pull request merged**